tests/quickstart.json

```diff
@@ -0,0 +1,190 @@
+{
+  "description": "This tests the 'building a smart contract' portion of the developer quickstart . It does not test forc-wallet or the contract deployment.",
+  "start_url": "guides/quickstart/building-a-smart-contract/",
+  "project_folder": "fuel-project",
+  "needs_wallet": true,
+  "steps": [
```

Comment:
It would be nice, to scan the steps directly from the page to avoid duplicate work. We can get this info using props on the component  `<TestAction />` it self.